### DISCUSSION TITLE
Entrega de practicas (Como y cuando)

### DISCUSSION THREAD

**Discussion by ByJ4Xx** [Launch-X-Latam/MisionFrontEnd#149] [Show and tell]
Solo queria hacer la aclaración aqui para todos los que estan preocupados acerca del tema.
En la sesión de hoy (5 de octubre) El Mission Commander nos explico que luego se abrira un "formulario" en el cual podremos subir nuestras practicas, no especifico una fecha en concreto por lo cual aun tienen tiempo para desarrollar la practica.
Ademas, el MC recomendó hacer un portafolio en el que se encuentren todas nuestras practicas para que a futuro podamos demostrar nuestra experiencia, capacidad y habilidad. 

Varios compañeros ya han creado repositorios de sus practicas, me parece que queda muy bien organizado y es un buen ejercicio para que se vayan familiarizando un poco con GitHub (Esto ultimo ya es mi opinión no me vayan a hacer caso del todo).
¡Suerte en sus practicas!

**Comment by snowingt** [comment]
Muchas gracias bro

**Reply by SantiagoCastellani** [reply]
Muchas gracias. 
Será que cada práctica de módulo va en un repo diferente? O todas las prácticas en el mismo repo?
Saludos...

**Comment by iam-leo** [comment]
Además esta bueno tener un respaldo en github y/o en la nube, ya que si le llegara a suceder algo a tu pc no perderías todo tu trabajo.

**Comment by AlvaroFierro** [comment]
Hola chicos, recuerden que hay una guia para los que aun andan medio perdidos esto esta en la discussion #247
